sessions/hack-time/README.md

```diff
@@ -0,0 +1,31 @@
+[IPFS Dev Meeting](../../README.md) | [Sessions](../README.md) | Hack Time
+
+# Sessions: Hack Time
+
+In pairs or small teams to tackle work units explicitly focused on code. **Aim for lots of pair programming.**
+
+- **Duration** 2 hours (take breaks as needed)
+- **Who Should Attend:** Anyone who wants to work on code with a partner or a small group  
+- **Who has to be there in order for it to work:** Many of the work packages involve knowledge from the maintainer who has worked on it the most. The tech leads will circulate around the various groups to answer questions and provide guidance.
+
+## Overview
+
+- **Objective:** Build people's working relationships while tackling known issues that will benefit from people doing synchronous work on them.  
+- **Activity:** In pairs or trios, spend 2 hours unpacking a work package together, designing solutions, pair programming, etc, then report back on the work you accomplished. _Especially focus on coding during these sessions._
+- **Outcomes:** Commit improvements & fixes and review or merge outstanding PRs related to important work packages.
+
+## Details
+
+
+# Proposed Hack Time Work Packages
+
+**Instructions**
+- **If you want to propose something to hack on**, add it here (via a Pull Request). Optionally use the [template](../_template.md) to provide a fuller description of the session.
+- **At the start of Hack Time** everyone will choose work packages using post-it notes: All of the work packages will be listed on pieces of paper on a wall. Each person gets one post-it note, puts their name on it, and attaches it to a work package. _Maximum of 3 people per work package._ If the work package is maxed out, you need to choose another one or negotiate swapping with someone else in the group.
```

Comment:
This is still saying `work packages` and describing max of 3 people. Hacking in trios is hard and inefficient. Effective pair programming is 2 only